api/src/Framework.php

```diff
@@ -1363,6 +1367,18 @@ protected function add_preferences_topmenu($type='prefs')
 					));
 				}
 				break;
+            case 'useraccount':
+                $config = Config::read('phpgwapi');
+                if ($config['own_account_acl'] && is_array($config['own_account_acl']) && count($config['own_account_acl'])>0){
```

Comment:
This should be: if (!empty($config['own_account_acl']) &&
To not raise a PHP Warning

Reply:
Done!